userspace/libsinsp/container.cpp

```diff
@@ -46,7 +46,8 @@ sinsp_container_manager::sinsp_container_manager(sinsp* inspector, bool static_c
 	m_static_container(static_container),
 	m_static_id(static_id),
 	m_static_name(static_name),
-	m_static_image(static_image)
+	m_static_image(static_image),
+	m_container_engine_mask(~0ULL)
```

Comment:
Shouldn't the default behavior be to enable all engines? This will require changes from the adopters for it to keep working as expected.

Reply:
It is. `~0ULL` has all bits set, and a set bit denotes an enabled engine, see e.g. https://github.com/falcosecurity/libs/pull/772/files#diff-0b9f04fd797523bf7465503b515f794972719a6c83c696f1b749b7ba1b9169c0R584

Of course, feel free to point out an obvious brainfart in my thinking ;)

Reply:
Never mind, I just got back from a long week end and my brain is taking longer than expected to boot up 😓